spec/requests/event_streams_spec.rb

```diff
@@ -162,4 +162,24 @@
       expect(response).to have_http_status(:forbidden)
     end
   end
+
+  describe 'OPTIONS /api/event_streams' do
+    it 'returns expected and additional attributes' do
+      MiqEventDefinitionSet.seed
+      options(api_event_streams_url)
+
+      expect_options_results(:event_streams)
+
+      body = response.parsed_body
+      expect(body["data"].keys).to eq(["timeline_events"])
+      expect(body["data"]["timeline_events"].keys.sort).to eq(%w[EmsEvent MiqEvent])
+      expect(body["data"]["timeline_events"]["EmsEvent"].keys.sort).to eq(%w[description group_levels group_names])
+      expect(body["data"]["timeline_events"]["EmsEvent"]["group_names"].keys.sort).to include("addition", "other")
```

Comment:
There's no need to test for all of them, just one from the event_handling and the default one "other"